### PR TITLE
[Juniper][QFX52xx] Platform fixes/enhancements

### PR DESCRIPTION
**Pull request by ciju-juniper**
This patch set implements the following:

 1) Fixing the issues while applying platform TxCTLE settings
    in sfputil.py for QFX5200
 2) Adding the support for transceiver dom threshold info in
    sfputil.py for both  QFX5210 & QFX5200 platforms
 3) Updating the sfputil.py for QFX5210 & QFX5200 platforms
 4) Adding a new platform specific command 'show_thresholds'
    to display the FAN dutycycle percentage for various
    temperature ranges (for both AFI & AFO QFX5200 systems).

Signed-off-by: Ciju Rajan K <crajank@juniper.net>

### CODE REVIEW

**Comment by ciju-juniper** [comment]
@jleveque @lguohan Now that all checks have passed, could you merge this patch? Thanks!

File: device/juniper/x86_64-juniper_qfx5200-r0/plugins/sfputil.py [hunk 1]
```diff
@@ -36,16 +36,23 @@
...
     from sonic_platform_base.sonic_sfp.sff8472 import sff8472InterfaceId
     from sonic_platform_base.sonic_sfp.sff8436 import sff8436Dom
     from sonic_platform_base.sonic_sfp.sff8436 import sff8436InterfaceId
+    from sonic_daemon_base.daemon_base import Logger
     from ctypes import create_string_buffer
 
 except ImportError as e:
     raise ImportError("%s - required module not found" % str(e))
 
+SYSLOG_IDENTIFIER = "xcvrd"
```
Comment (jleveque) [inline device/juniper/x86_64-juniper_qfx5200-r0/plugins/sfputil.py:51]: Why syslog identifier of "xcvrd"? Identifier should be specific to your plugin.

File: device/juniper/x86_64-juniper_qfx5210-r0/plugins/sfputil.py [hunk 2]
```diff
@@ -5,13 +5,15 @@
...
+    import io
     import string
     from ctypes import create_string_buffer
-    # sys.path.append('/usr/local/bin')
-    # import sfp_detect
+    from sonic_daemon_base.daemon_base import Logger
 except ImportError, e:
     raise ImportError (str(e) + "- required module not found")
 
+SYSLOG_IDENTIFIER = "xcvrd"
```
Comment (jleveque) [inline device/juniper/x86_64-juniper_qfx5210-r0/plugins/sfputil.py:15]: Why syslog identifier of "xcvrd"? Identifier should be specific to your plugin.

File: device/juniper/x86_64-juniper_qfx5200-r0/plugins/sfputil.py (hunk 1 above)
Comment (ciju-juniper) [reply device/juniper/x86_64-juniper_qfx5200-r0/plugins/sfputil.py]: Updated

File: device/juniper/x86_64-juniper_qfx5210-r0/plugins/sfputil.py (hunk 2 above)
Comment (ciju-juniper) [reply device/juniper/x86_64-juniper_qfx5210-r0/plugins/sfputil.py]: Updated.

**Comment by ciju-juniper** [comment]
@jleveque I've addressed your comments. Please see if this can be merged now. Thanks!

**Pull request merged**